internal/buildscripts/packaging/fpm/common.sh

```diff
@@ -16,7 +16,7 @@ SERVICE_GROUP="splunk-otel-collector"
 
 OTELCOL_INSTALL_PATH="/usr/bin/otelcol"
 CONFIG_REPO_PATH="$REPO_DIR/cmd/otelcol/config/collector/agent_config.yaml"
-CONFIG_INSTALL_PATH="/etc/otel/collector/splunk_config_linux.yaml"
+CONFIG_INSTALL_PATH="/etc/otel/collector/gateway_config.yaml"
```

Comment:
For the default config file in the deb/rpm packages, maybe we just package the file as `config.yaml` instead of `gateway_config.yaml`. We'll then need to add the `SPLUNK_CONFIG=/etc/otel/collector/config.yaml` env var to the collector service environment file to override whatever is the default in main.go.  I can make the necessary changes after this PR is merged.